### ISSUE TITLE
Allow setting different values for jumpInterval forward and jumpInterval backward

### ISSUE THREAD

**Issue by mitchdowney** [DoubleSymmetry/react-native-track-player#1174]
**Is your feature request related to a problem? Please describe.**
Currently in Podverse the jump backward button changes the position by 10 seconds, and the jump forward button changes by 30 seconds.

While this works in our in-app media player, we cannot set different time values for the lock screen jump buttons, because react-native-track-player only allows us to configure a jumpInterval value that is used by both jump buttons.

It's a better UX in podcast apps to provide 2 different jump values, so that the user can more quickly navigate to the section they want to listen to. (Many users requested this feature from us.)

I see in this issue https://github.com/DoubleSymmetry/react-native-track-player/issues/284 @Pczek requested the ability to set a jumpIntervalForward and jumpIntervalBackward property.

**Describe the solution you'd like**
1) On the lock screen, press the jump backward button to jump backward X seconds.
2) On the lock screen, press the jump forward button to jump forward Y seconds.

**Describe alternatives you've considered**
I don't think there's an alternative to accomplish this on lock screens.

**Additional context**

It'd be awesome for podcast apps if react-native-track-player could support two different jump button values :)

Thanks for the awesome library. It's extremely valuable for the [Podverse mobile app](https://github.com/podverse/podverse-rn).

**Comment by dcvz** [comment]
Thanks for sharing your use case! We've actually planned to build this in the past.

At the moment we're focusing on releasing our version 2.0.0 of this library. I might be able to get this change done fairly soon but it would only be available in our 2.0.0 betas.

How heavily is Podverse relying on IDs for queue management?

**Comment by dcvz** [comment]
@mitchdowney this has been added in #1177 and will be part of the next pre-release: `2.0.0-rc15` going out later today!

**Comment by mitchdowney** [comment]
Fantastic! Thanks for the quick response!

> How heavily is Podverse relying on IDs for queue management?

We currently use the URL of media files we are loading into the player as their IDs.

---

Not sure if this is relevant, but FYI our queue management process is fairly complicated. I haven't touched it for over a year, but at one point, we were seeing differences between react-native-track-player queue and event handling between iOS and Android devices. To work around the issues and ensure consistency, we started clearing the queue, and adding everything back into to the queue whenever an item is added/moved/removed from the queue.

In other words, we have our own queue management layer, and whenever the react-native-track-player queue needs to update, we clear the whole thing (except for the currently playing item) and sync it with our own queue service.

Anyway, maybe that's a problem with how we've been using react-native-track-player. It's been a long time since I touched the queue handling. Just wanted to share that since you brought up queue IDs.

**Comment by dcvz** [comment]
@mitchdowney try to update the lib to the latest pre-release (and take a look at my "release notes" in that ticket)! I asked about IDs for queue management because we've updated all methods around the queue to work based on index instead of id: `seekTo(index)`, `add(track, index)`, etc.

======

I'm really curious about what problems you had around the queue -- the next plan is to revisit the queue management utilities and make them more performant so that would be super interesting information for me.
